mineral-ui/mineral-ui
# [Text] New component

### Description
* New component Text
* Add Text demo & examples
* Update website content to use Text 
    * Update Markdown and Typography page
    * Did not update site headings
* Add spec & snapshots


### Motivation and context


Closes #158 

### Screenshots, videos, or demo, if appropriate


https://158-text--mineral-ui.netlify.com/components/text

### How to test


1. `cd mineral-ui && npm start`
2. Navigate to Text > confirm that it matches the spec and confirm no spelling errors
3. Navigate to the Typography page & confirm it matches the spec

### Types of changes

- New feature (non-breaking change which adds functionality)

### Checklist




### How does this PR make you feel?

![Text gif](https://media.giphy.com/media/R3IxJW14a3QNa/giphy.gif)
